controllers/adminController.js

```diff
@@ -0,0 +1,61 @@
+const userRoleManager = require('../lib/modelManagers/userRoleManager');
```

Comment:
` controllers/adminController.js ` Please, I would love to know why we have an adminController.

Reply:
I also would love to hear your reasons for not having controllers to perform the following operations;
1. Role deletions
1. Role updates
1. Fetching Roles in the system
1. Fetch a particular role

Reply:
I have fixed it.